### PR TITLE
omits empty organisms

### PR DESCRIPTION
**Pull request by nathandunn**
adds the omitEmptyOrganism option per: https://github.com/GMOD/Apollo/pull/1800#issuecomment-342969601

### CODE REVIEW

**Comment by nathandunn** [comment]
@deepakunni3  One more.

File: docs/web_services/examples/shell/loadUsers.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ if [[ ! -n "$url" || ! -n "$username" || ! -n "$password" ]]; then
 fi
 
 echo "Load all users"
-curl  -i -X POST -H 'Content-Type: application/json' -d "{'username': '${username}', 'password': '${password}'}" "${url}/user/loadUsers"
+curl  -i -X POST -H 'Content-Type: application/json' -d "{'omitEmptyOrganisms':Grue,'username': '${username}', 'password': '${password}'}" "${url}/user/loadUsers"
```
Comment (deepakunni3) [inline docs/web_services/examples/shell/loadUsers.sh]: `Grue` -> `true`

**Comment by nathandunn** [comment]
@deepakunni3  Fixed the typo

**Comment by deepakunni3** [comment]
Also, it looks like `scripts/load_users.sh` is outdated. Perhaps remove in favor of `docs/web_services/examples/shell/loadUsers.shloadUsers.sh`?

**Comment by nathandunn** [comment]
Yeah, I didn't realize that other one even existed.   Yes, please remove the one under scripts in favor the web_servces one.  thx

**Comment by deepakunni3** [comment]
@nathandunn Done. Tested and it works as intended. 👍

**Pull request merged**